src/components/debug/ContextDebugger.tsx

```
@@ -0,0 +1,440 @@
+  const contextForEmbedded = sdkContext || null;
+  const [expandedSections, setExpandedSections] = useState<Set<string>>(new Set(['summary']));
+  const [copySuccess, setCopySuccess] = useState(false);
+  
+  // Get raw SDK instance for quickAuth access
+  const sdkContextState = useContext(MiniAppSdkContext);
+  const rawSdkInstance = sdkContextState?.sdk;
+  
+  // Get authenticator manager (may not be available if not logged in)
+  let authenticatorManager: ReturnType<typeof useAuthenticatorManager> | null = null;
+  try {
+    authenticatorManager = useAuthenticatorManager();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)